src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsresettlementpassportapi/resource/ResettlementAssessmentController.kt

```diff
@@ -275,11 +275,13 @@ class ResettlementAssessmentController(
     assessmentType: ResettlementAssessmentType,
     @RequestParam("useNewDeliusCaseNoteFormat")
     useNewDeliusCaseNoteFormat: Boolean = false,
+    @RequestParam("declaration")
+    declaration: Boolean = false,
```

Comment:
This declaration box is on the check your answers page so we need to add this to the /complete endpoint